modules/swagger-codegen/src/main/resources/r/api.mustache

```diff
@@ -1,56 +1,94 @@
 {{>partial_header}}
 {{#operations}}
+#' @title {{baseName}} operations
+#' @description {{importPath}}
+#'
+#' @field path Stores url path of the request.
+#' @field apiClient Handles the client-server communication.
+#' @field userAgent Set the user agent of the request.
+#'
+#' @importFrom R6 R6Class
+#'
+#' @section Methods:
+#' \describe{
+{{#operation}}
+#'
+#' {{operationId}} {{summary}} 
+#'
+{{/operation}}
+#' }
+#' 
+#' @export
 {{classname}} <- R6::R6Class(
   '{{classname}}',
   public = list(
     userAgent = "{{#httpUserAgent}}{{{.}}}{{/httpUserAgent}}{{^httpUserAgent}}Swagger-Codegen/{{{packageVersion}}}/r{{/httpUserAgent}}",
-    basePath = "{{{basePath}}}",
-    initialize = function(basePath){
-      if (!missing(basePath)) {
```

Comment:
@brnleehng thanks for the PR. Does it mean developers (users of the SDK) can no longer use a different base path during runtime when constructing the class?

Reply:
Ah please ignore. You've moved "basePath" to apiClient.